assets/css/rtl.scss

```diff
@@ -1,5 +1,5 @@
 // Algolia search
-.algolia-wrapper {
+.algolia-wrapper.rtl {
```

Comment:
You can directly use the setting from the HTML tag
```suggestion
html[dir='rtl'] .algolia-wrapper {
```

Reply:
I'll apply that. Didn't know about this feature